src/Entities/Transaction.php

```diff
@@ -122,6 +144,48 @@ enum: [
         );
         $this->paymentMethodDetails = $paymentMethodDetails ?: new PaymentMethodDetails();
 
+        if ($payoutSystem) $this->payoutSystem = $this->createSimpleField(
+            propertyName: 'payout_system',
+            value: $payoutSystem
+        );
+
+        if ($transactionType) $this->transactionType = $this->createSimpleField(
+            propertyName: 'transaction_type',
+            value: $transactionType
+        );
+
+        if ($mscVatPercentage) $this->mscVatPercentage = $this->createSimpleField(
+            propertyName: 'msc_vat_percentage',
+            value: $mscVatPercentage
+        );
+
+        if ($mscVatClass) $this->mscVatClass = $this->createSimpleField(
+            propertyName: "msc_vat_class",
+            value: $mscVatClass
+        );
+
+        if ($mscVatAmount) $this->mscVatAmount = $this->createSimpleField(
+            propertyName: 'msc_vat_amount',
+            value: $mscVatAmount
+        );
```

Comment:
It is not matter, but propertyNames in single quotes. Like 'msc_vat_percentage'
But on line 163:` propertyName: "msc_vat_class"` in double quotes

Reply:
OK